examples/linear_elasticity/linear_elastic_probes.py

```diff
@@ -138,22 +138,22 @@ def post_process(out, problem, state, extend=False):
         plt.xlabel('probe %s' % label, fontsize=8)
         plt.legend(loc='best', prop=fm.FontProperties(size=10))
 
-        sym_indices = [0, 4, 8, 1, 2, 5]
         sym_labels = ['11', '22', '33', '12', '13', '23']
 
         plt.subplot(312)
         pars, vals = probe(probe_name, 'cauchy_strain')
-        for ii, ic in enumerate(sym_indices):
-            plt.plot(pars, vals[:,ic], label=r'$e_{%s}$' % sym_labels[ii],
+        print(vals.shape)
```

Comment:
debug print?

Reply:
Yes, forgotten debug print. I will remove it.